main.cpp

```diff
@@ -330,23 +330,52 @@ void loop() {
 }
 
 int main() {
-    srand (time(NULL));
-    int c;
-    cout << "Wpisz rozmiar generowanej planszy lub 0, zeby pobrac plansze z pliku: ";
-    cin >> c;
-    cout << "Pobrano " << c << '\n';
     try {
-        if(c) {
-            generateMap(c,c);
-            loop();
-        } else {
+    cout<<"Witaj w naszej amatorskiej grze 'Labirynt'."<<endl<<endl<<endl;
+    cout<<"----------MENU----------"                   <<endl;
+    cout<<"| Wybierz:             |"                   <<endl;
+    cout<<"| 1. - Kampania        |"                   <<endl;
+    cout<<"| 2. - Losowa mapa     |"                   <<endl;
+    cout<<"| 3. - Wyswietl Tworcow|"                   <<endl;
+    cout<<"------------------------"                   <<endl;
```

Comment:
Inaczej sobie wyobrażałem ładne menu, ale powiedzmy, że w #43 lepiej to zrobisz. :)